lib/generators/manageiq/plugin/templates/bin/setup

```diff
@@ -2,8 +2,14 @@
 require 'pathname'
 
 gem_root = Pathname.new(__dir__).join("..")
+spec_manageiq = gem_root.join("spec/manageiq")
 
-unless gem_root.join("spec/manageiq").exist?
+if ENV.key?("MANAGEIQ_REPO")
+  puts "== Symlinking spec/manageiq to #{ENV["MANAGEIQ_REPO"]}"
+
+  spec_manageiq.rmtree if spec_manageiq.exist?
+  system "ln -s #{ENV["MANAGEIQ_REPO"]} spec/manageiq"
```

Comment:
Only concern here is whether we should fully qualify things in case things are not fully qualified and/or not run from the root of the directory?

```ruby
system "ln -s #{File.expand_path(ENV["MANAGEIQ_REPO"])} #{File.expand_path("../spec/manageiq", __dir__)}"
```

Reply:
Okay switched it to use fully-qualified paths, and tested `MANAGEIQ_REPO=../manageiq bin/setup` as well as `cd bin; MANAGEIQ_REPO=../../manageiq ./setup`